sdk/identity/azure-identity/src/chained_token_credential.cpp

```diff
@@ -139,5 +125,5 @@ AccessToken ChainedTokenCredential::GetToken(
     }
   }
 
-  throw AuthenticationException("Failed to get token from ChainedTokenCredential.");
+  throw AuthenticationException("Failed to get token from " + CredentialName + '.');
```

Comment:
nit: Call the public `GetCredentialName()` to reduce uses of private impl detail.